### PR TITLE
Spelling Correction

### PR DESCRIPTION
**Pull request by HobbesE**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 print("Hello to the whole wolrd!")
 
-print("Goodbye to the whole wolrd!")
\ No newline at end of file
+print("Goodbye to the whole world!")
```
Comment (zacharywilkins) [inline app.py:3]: Great catch!

**Review by zacharywilkins — CHANGES_REQUESTED** [review]
Great catch, Hobbes! Before I approve this PR, can you see if there are any other typos in this file?

File: app.py (hunk 1 above)
Comment (HobbesE) [reply app.py:3]: Oh, got it! Thanks!

**Review by zacharywilkins — APPROVED** [review]
LGTM!

**Pull request merged**